legacy/src/webhooks/ntfy.ts

```
@@ -0,0 +1,314 @@
+    if (value.includes(',') || value.includes(';')) {
+      return `"${value.replace(/"/g, '\\"')}"`;
+    }
+    return value;
+  }
+
+  /**
+   * Sanitize title by removing emojis and non-ASCII characters
+   */
+  private sanitizeTitle(title: string | undefined | null): string {
+    if (!title) return '';
+    return title.replace(/[^\x00-\x7F]/g, '');
```

Biome (2.1.2)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)